lib/mote_sms/message.rb

```diff
@@ -95,8 +100,20 @@ def to(*numbers)
     #
     # Returns result of transport#deliver.
     def deliver(options = {})
+      Kernel.warn 'Message#deliver is deprecated and will be removed from MoteSMS. Please use #deliver_now'
+      deliver_now options
+    end
+
+    def deliver_now(options = {})
+      Kernel.warn 'options[:transport] in Message#deliver_now is deprecated and will be removed from MoteSMS' if options[:transport]
       transport = options.delete(:transport) || self.transport || MoteSMS.transport
       transport.deliver(self, options)
     end
+
+    def deliver_later(options = {})
+      return Kernel.warn 'options[:transport] is not supported in Message#deliveer_later' if options.delete(:transport)
+      raise 'huhuhu' unless defined?(ActiveJob)
+      DeliveryJob.perform_later @from, @to, @body, options
```

Comment:
not sure the `raise` is necessary as it will fail when using DeliveryJob, not? maybe we should also make this opt-in, i.e. extend this into a module which is only loaded when we can load active_job?